lib/iex/lib/iex/parser.ex

```diff
@@ -0,0 +1,36 @@
+defmodule IEx.Parser do
```

Comment:
There is no need for this to be a separate module. You can keep this function in the `IEx.Evaluator` module as long as it is public with `@doc false`. :)